scripts/exregional_save_restart.sh

```diff
@@ -112,28 +112,31 @@ restart_prefix=${save_yyyy}${save_mm}${save_dd}.${save_hh}0000
 if [ ! -r ${nwges_dir}/INPUT/gfs_ctrl.nc ]; then
   cp_vrfy $run_dir/INPUT/gfs_ctrl.nc ${nwges_dir}/INPUT/gfs_ctrl.nc
   if [ -r ${run_dir}/INPUT/coupler.res ]; then  # warm start
-    if [ "${IO_LAYOUT_Y}" == "1" ]; then
-      for file in ${filelistn}; do
-        cp_vrfy $run_dir/INPUT/${file} ${nwges_dir}/INPUT/${file}
-      done
-    else
-      for file in ${filelistn}; do
-        for ii in ${list_iolayout}
-        do
-          iii=$(printf %4.4i $ii)
-         cp_vrfy $run_dir/INPUT/${file}.${iii} ${nwges_dir}/INPUT/${file}.${iii}
+    if [ "${DO_SAVE_INPUT}" = TRUE ]; then
```

Comment:
I don't understand why line 115 is added. Saving restart files to nwges is required for continuous cycles (warm start) and doesn't need a switch, right?

Reply:
We only need INPUT/gfs_ctrl.nc and the files under RESTART/ for warm start cycles, not the other files under INPUT, right? What are these files under INPUT/ being used for?

Reply:
STMP will not save anything and will be removed immediately following the completion of a cycle in operation. So we need to save all IC/BC into NWGES and any cycle gets them from NWGES (not STMP).

Reply:
For all our current test, STMP is saved for a day. Operation is another story.

Reply:
We are talking about the files under nwges. So maybe we should set the default value of 'DO_SAVE_INPUT" to be "TRUE" and for ensemble runs we set it to be FALSE in config.sh. What do you think? @hu5970 @guoqing-noaa

Reply:
@hu5970 I am confused by your comments. So what tasks will save the 1h fcst to nwges for the next cycle?

Reply:
@chunhuazhou I would suggest limiting this switch logic inside the script if ensembles don't need to save RESTARTS as end users don't need to decide when to save ICs for the next cycle.

Reply:
save_restart task saves the 1-h forecast (restart files) for warm start for the next cycle. These files are saved to nwges/.../fcst_fv3lam/RESTART
This PR is about the files under nwges/.../fcst_fv3lam/INPUT, under which all the files from stmp/.../fcst_fv3lam/INPUT are being saved here. 
As @hu5970 commented, only gfs_ctrl under nwges/.../fcst_fv3lam/INPUT is being used for the next cycle. For our ensemble DA runs, we have a task to save the DA analysis in preparation for the ensemble forecast runs. In this case, we have duplicate copies of the files under nwges/.../fcst_fv3lam/INPUT for all the members and nwges/.../fcst_fv3lam/DA_OUTPUT for only the first 9 members.
This PR is to address this and avoid the redundant copy.

Reply:
OK, I did not realize that in save_restart.sh, it saves both INPUT and RESTART. Thanks for the clarification. It looks fine to me now.

Reply:
Thanks for the comments @guoqing-noaa 
I changed the default value of "DO_SAVE_INPUT" to "TRUE" so by default it won't change anything from what we have been running. But for ensemble DA cycling, we turn this off. Please take a look and see if the changes make sense to you @hu5970 @guoqing-noaa